src/index.tsx

```diff
@@ -2,11 +2,13 @@ import { StrictMode } from "react";
 import { render } from "react-dom";
 import { BrowserRouter } from "react-router-dom";
 import "./index.css";
-import { App } from "./App";
+import { App } from "./components/App";
 import { reportWebVitals } from "./reportWebVitals";
 
-// Ugly patch to try to override the basename from outside
-const basename = (window as any).basename || undefined;
+const basename =
+  (window as any)["editors-webapp-configuration"]?.basename ||
+  (window as any).basename || // TODO: remove this option
```

Comment:
It is to keep backward compatibility with https://github.com/MakingSense/doppler-swarm/pull/581/files#diff-fefb0937f1cb984600200829d4930039926419dc02c3b4b0b3ca4b222f9b65e6L40